public/docs/ts/latest/guide/index.jade

```diff
@@ -1,109 +1,111 @@
 include ../../../../_includes/_util-fns
 
+figure
+img(src="/resources/images/devguide/intro/people.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
 :marked
   The Developer Guide is a practical guide to Angular for experienced programmers who
   are building client applications in HTML and JavaScript.
-figure
-  img(src="/resources/images/devguide/intro/people.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
 :marked
-  We are on a journey together to understand how Angular works and, more importantly,
-  how to make it work for us. We look at our application requirements and we see problems to solve.
+  In this guide, we will go on a journey together to understand how Angular works and, more importantly,
+  how to make it work for us. We'll look at many typical application requirements and how Angular
+  provides us solutions, such as:
   <br clear="all">
-  
-  * How do we get data onto the screen and handle user interactions?
-  * How do we organize our code into manageable, cohesive chunks of functionality that work together?
+
   * What are the essential Angular building blocks and how do they help?
+  * How do we get data onto the screen and handle user interactions, such as click events or forms?
+  * How do we organize our code into manageable, cohesive chunks of functionality that work together?
   * How do we minimize routine, mechanical coding in favor of declarative, higher level constructs without losing control?
-  
-  This chapter begins the journey.
-  
+
+  This guide is a collaborative effort by members of the Angular community, and is still in development. We've done
```

Comment:
We do not want to apologize for incompleteness. It will always be incomplete.

What is the value of "collaborative effort"? We're trying to make it sound like a single author.

Why talk about our "best effort"? Why would we give less than our best effort?